CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Adding a new modeling command
+
+ - In `each_command.rs` add your new `struct MyNewCommand`
+ - In `def_enum.rs` add a new variant of `ModelingCmd` with your type, e.g. `MyNewCommand(MyNewCommand)`.
+ - If your command responds with data:
+   - In `output.rs`, add a `struct MyNewCommand` following the existing examples.
+   - Then scroll to the end of the file and `impl<'de> ModelingCmdOutput<'de> for MyNewCommand {}`
+ - In `impl_traits.rs` follow the examples to implement `ModelingCmdVariant<'de>` for your type `MyNewCommand` using either the `impl_variant_output!` or the `impl_variant_empty!` macro. If your command responds with data, use the former. If your command has no response, use the latter.
```

Comment:
I'm not sure if this is still necessary, but I also had to modify ok_response.rs with 
/// The response from `MyNewCommand`
MyNewCommand(output::MyNewCommand)

Reply:
Ah yeah good point